crates/trigger/src/cli/initial_kv_setter.rs

```diff
@@ -0,0 +1,42 @@
+use anyhow::Context as _;
```

Comment:
This file might seem to be an unnecessary change, but it was the easiest way to continue to support setting key-value pairs on start up. I imagine even if we don't accept this PR, we'll want to implement this change.